suite/performance/vrf_soak_test.go

```diff
@@ -68,7 +65,6 @@ var _ = Describe("VRF soak test @soak-vrf", func() {
 				defaultNetwork.Client,
 				defaultNetwork.Wallets,
 				defaultNetwork.Deployer,
-				adapter,
```

Comment:
Was this just not being used in the VRF tests?

Reply:
yes